src/boot/bootctl.c

```diff
@@ -1130,6 +1141,15 @@ static int parse_argv(int argc, char *argv[]) {
                 case ARG_GRACEFUL:
                         arg_graceful = true;
                         break;
+                case ARG_JSON_FORMAT:
```

Comment:
please use the json_parse_cmdline_parameter_and_warn helper - see src/dissect/dissect.c for an example